.github/workflows/cp-beta.yaml

```diff
@@ -0,0 +1,124 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: cherrypick
```

Comment:
Can we have a more descriptive name?

1. Cherry-pick to beta branch
2. Create cherry-pick to branch
3. etc

Reply:
Sure, renamed to `Cherry-pick Labeled PR to Beta Branch`